src/analyzer/protocol/ssl/SSL.cc

```diff
@@ -16,12 +84,20 @@ SSL_Analyzer::SSL_Analyzer(Connection* c)
 	interp = new binpac::SSL::SSL_Conn(this);
 	handshake_interp = new binpac::TLSHandshake::Handshake_Conn(this);
 	had_gap = false;
+	c_seq = 0;
+	s_seq = 0;
+	// FIXME: should this be initialized to nullptr?
```

Comment:
yup :)

Reply:
Also - for extra points - we try to use intrusive-pointers in new code. 

https://github.com/zeek/zeek/wiki/IntrusivePtr-Overview